src/Microsoft.DotNet.Darc/src/DarcLib/IGitRepoExtension.cs

```diff
@@ -19,7 +20,11 @@ public static string GetDecodedContent(this IGitRepo gitRepo, string encodedCont
             try
             {
                 byte[] content = Convert.FromBase64String(encodedContent);
-                return Encoding.UTF8.GetString(content);
+                using (var str = new MemoryStream(content, false))
+                using (var reader = new StreamReader(str))
```

Comment:
Can you put a little note in here as to why this was implemented this way vs. the more obvious UTF8.GetString?

Reply:
sure

Reply:
I hope no one uses some other none UTF8 encoding. :-) I don't think I'd mind removing the BOM's? We should only be doing this to file that we "control", right?

Reply:
yeah, we 'control' it.  I think removing the BOM's is the right thing.  I hate BOMs